server/plugin/command.go

```diff
@@ -15,37 +15,41 @@ import (
 )
 
 const (
-	featureIssueCreation = "issue_creations"
-	featureIssues        = "issues"
-	featurePulls         = "pulls"
-	featurePullsMerged   = "pulls_merged"
-	featurePullsCreated  = "pulls_created"
-	featurePushes        = "pushes"
-	featureCreates       = "creates"
-	featureDeletes       = "deletes"
-	featureIssueComments = "issue_comments"
-	featurePullReviews   = "pull_reviews"
-	featureStars         = "stars"
-	featureReleases      = "releases"
+	featureIssueCreation      = "issue_creations"
+	featureIssues             = "issues"
+	featurePulls              = "pulls"
+	featurePullsMerged        = "pulls_merged"
+	featurePullsCreated       = "pulls_created"
+	featurePushes             = "pushes"
+	featureCreates            = "creates"
+	featureDeletes            = "deletes"
+	featureIssueComments      = "issue_comments"
+	featurePullReviews        = "pull_reviews"
+	featureStars              = "stars"
+	featureReleases           = "releases"
+	featureDisccusions        = "discussions"
+	featureDiscussionComments = "discussion_comments"
 )
 
 const (
 	PerPageValue = 50
 )
 
 var validFeatures = map[string]bool{
-	featureIssueCreation: true,
-	featureIssues:        true,
-	featurePulls:         true,
-	featurePullsMerged:   true,
-	featurePullsCreated:  true,
-	featurePushes:        true,
-	featureCreates:       true,
-	featureDeletes:       true,
-	featureIssueComments: true,
-	featurePullReviews:   true,
-	featureStars:         true,
-	featureReleases:      true,
+	featureIssueCreation:      true,
+	featureIssues:             true,
+	featurePulls:              true,
+	featurePullsMerged:        true,
+	featurePullsCreated:       true,
+	featurePushes:             true,
+	featureCreates:            true,
+	featureDeletes:            true,
+	featureIssueComments:      true,
+	featurePullReviews:        true,
+	featureStars:              true,
+	featureReleases:           true,
+	featureDisccusions:        true,
```

Comment:
Typo in variable name here as well.

Reply:
Updated in [95e2155](https://github.com/mattermost/mattermost-plugin-github/pull/808/commits/95e2155d254a1fc6ed29cb819f0cd511ddd0a0c5)

Reply:
@elewis787 still has a typo

```suggestion
	featureDiscussions:        true,
```

Reply:
Sorry about that - I committed the suggestion and updated the subscription check to use the updated spelling.